recipe/meta.yaml

```
@@ -1,45 +1,45 @@
 {% set name = "streamlit" %}
-{% set version = "1.32.0" %}
+{% set version = "1.37.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 65bf22e190de973b910c0b127b33d80d31c5275ed891c6fde740de46e5e05323
+  sha256: bc7e3813d94a39dda56f15678437eb37830973c601e8e574f2225a7bf188ea5a
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   entry_points:
     - streamlit = streamlit.web.cli:main
-  skip: true  # [s390x or py<38]
+  skip: true  # [s390x or py<=38]
```

Based on `Requires-Python: >=3.8, !=3.9.7`?:

```suggestion
  skip: true  # [s390x or py<38]
```